#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_typo_fix.dpatch by Martijn van Brummelen <martijn@brumit.nl>
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes several small typo's (Closes: 498099)

@DPATCH@
diff -urNad acx100-20080210~/script/fetch_firmware.sh acx100-20080210/script/fetch_firmware.sh
--- acx100-20080210~/script/fetch_firmware.sh	2008-02-10 21:06:42.000000000 +0100
+++ acx100-20080210/script/fetch_firmware.sh	2009-11-05 12:46:37.000000000 +0100
@@ -15,8 +15,8 @@
 find_driver_dir()
 {
   ACXDIR=`pwd`
-  [ -f $ACXDIR/scripts/start_nets.sh ] && return
-  if [ -f $ACXDIR/../scripts/start_net.sh ]; then
+  [ -f $ACXDIR/script/start_net.sh ] && return
+  if [ -f $ACXDIR/../script/start_net.sh ]; then
     ACXDIR="$ACXDIR/.."
     return
   fi
diff -urNad acx100-20080210~/script/inject_kernel_tree.sh acx100-20080210/script/inject_kernel_tree.sh
--- acx100-20080210~/script/inject_kernel_tree.sh	2008-02-10 21:06:42.000000000 +0100
+++ acx100-20080210/script/inject_kernel_tree.sh	2009-11-05 12:47:55.000000000 +0100
@@ -31,7 +31,7 @@
 }
 
 print "- Checking acx sources...\n";
-my $kernel_help = "$from_dir/scripts/kernel_help";
+my $kernel_help = "$from_dir/script/kernel_help";
 if (!-e "$from_dir/src") {
     print "*** $from_dir/src doesn't exist!\n";
     exit 1;
